docs/build/getting-started/making-first-request.md

```diff
@@ -1 +1,200 @@
-# Making Your First Request
\ No newline at end of file
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
+
+# Making Your First Request
```

Comment:
hmm, is there a reason why we're focusing solely on Lit Actions here? I might argue that LA's are our _most_ complex offering and perhaps we want to start simple? I'd say the base functionality is both generating a signature (we could use pkpSign endpoint instead of Lit Action signing) and encrypting + decrypting something. 

the same steps would apply (up to 'generating session sigs') but then we could branch off into generating a signature OR encrypt + decrypt a string? What do you think?

Reply:
Lit Actions are what Wyatt suggested in the outline for these docs. I'd say the existing Lit Action example complexity is alright, considering it doesn't involve a PKP or signing, and just displays executing code in a decentralized environment. 

With encryption/decryption, I'd say it is encapsulated more within Lit Actions than visa versa (you can decrypt within a Lit Action). Because we use Lit Actions to cover a broader scope of operations, while they're more complex, I think getting developers familiar with them is a high priority.